packages/cli/src/utils/cli.ts

```diff
@@ -187,3 +194,14 @@ export function humanizeRequirements(requirements: LockedGoldRequirements) {
   })
   return { requiredCelo, requiredDays }
 }
+
+// TODO this will have to be refactored not to use BigNumber
+// once we drop its usage
+function bigintToExponential(value: bigint) {
+  // as per the check in `toStringValueMapRecursive`
+  if (value <= 1000n) {
```

Comment:
lets make a const that us shared between the 2 places instead of relying on comments

Reply:
actually do we need this if at all?  just have the function always return exponential that simplifiess things

Reply:
from what i see its only called if the value is 1000 anyway